index.js

```diff
@@ -0,0 +1,3 @@
+window.Ps = require('perfect-scrollbar');
```

Comment:
Well, I thought about this a bit. I could require it in the src file itself and avoid this window stuff. I am just not sure the best way to go about it.